app/2.0/docs/tools/create-app-polymer-cli.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Create an application project with the Polymer CLI
+---
+
+<!-- toc -->
+
+Polymer CLI supports initializing a project folder with one of several application templates.  The 
+CLI comes with a `basic` template, which is the most basic starting point for a Polymer-based 
+application, as well as others that introduce more complex layout and application patterns.
+
+This chapter teaches you more about `basic` app projects.  See [Polymer App Toolbox 
+templates](../../toolbox/templates) for more details on other templates.
+
+## App project architecture {#app-architecture}
+
+Polymer CLI is designed for apps that follow the [app shell 
+architecture](https://developers.google.com/web/updates/2015/11/app-shell).
+
+There are fundamental concepts of the app shell architecture that you should understand before 
+creating your app project with Polymer CLI: the entrypoint, the shell, and fragments. See [App 
+structure](../../toolbox/prpl#app-structure) from the App Toolbox docs for an in-depth overview of 
+these concepts.
+
+## Set up basic app project {#basic-app}
+
+Follow the steps below to get your `basic` app project set up.
+
+1.  Create a directory for your app project.
+
+    <pre><code>mkdir <var>app</var>
+    cd <var>app</var></code></pre>
+
+    Where <code><var>app</var></code> is the name of your project directory.
+
+1.  Initialize your app. Polymer CLI asks you a few questions
+    as it sets up your app.
+
+        polymer init
+
+1.  Select `polymer-2-application`.
+
+1.  Enter a name for your app. Defaults to the name of the current directory.
```

Comment:
I agree that it would be unreasonable to expect the team to remember to update the CLI docs with these changes. I will maintain this documentation :)

For now I'm happy to leave these steps here with explanations for each. For instance, the user is asked to give a name to their "main element" and as you've noted below, the concept of a "main element" is not clear without an explanation.